packages/core/src/Asset/util/metadatumToCip25.ts

```diff
@@ -23,8 +23,8 @@ const asStringArray = (metadatum: Cardano.Metadatum | undefined): string[] | und
     if (result.some((str) => typeof str === 'undefined')) {
       return undefined;
     }
-    // Based on the CIP25: base64-encoded fields spec
-    if (result[0]?.startsWith('data:')) {
+    // Based on the CIP-25 image validation specs
+    if (result[0]?.startsWith('data:') || result[0]?.startsWith('ipfs://')) {
```

Comment:
I believe this behavior doesn't depend on protocol (same with https)
```suggestion
```

Reply:
Yes, I excluded it because the image could be 
`["https://cardano.org/favicon-32x32.png", "https://cardano.org/favicon-10x10.png"]` which is a valid one.

Haven't seen like `["https://cardano.org/favicon", "-10x10.png"]` 
 @mkazlauskas

Reply:
bfc177bd8b2c90e2256a12d122edd4d506ca436d